src/FilterContext.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from "react";
+import { createContext } from "react";
+import { ProductsDataBackUp } from "./ProductsDataBackUp";
+
+interface Counter {
+    number: number;
+  }
+
+export const maxPriceContext = createContext<[Counter, React.Dispatch<React.SetStateAction<Counter>>] | undefined>(undefined);
+
+
+const FilterContext = () => {
+    // const [maxPrice, setMaxPrice] = useState<number | undefined>();
```

Comment:
??????????????????????????????????????????????????

Reply:
if its about the undefined i fixed it :(